### PR TITLE
CHAD-4088 Remove legacy "status" variable from SmartSense Multi and S…

### PR DESCRIPTION
**Pull request by greens**
…martSense Garage Door

### CODE REVIEW

File: devicetypes/smartthings/smartsense-multi.src/smartsense-multi.groovy [hunk 1]
```diff
@@ -301,17 +288,6 @@ private List getContactResult(part, description) {
 			displayed:false
```
Comment (tpmanley) [inline devicetypes/smartthings/smartsense-multi.src/smartsense-multi.groovy]: This should be changed to `true`

Comment (tpmanley) [reply devicetypes/smartthings/smartsense-multi.src/smartsense-multi.groovy]: Or rather not set at all so it is set automatically

File: devicetypes/smartthings/smartsense-garage-door-multi.src/smartsense-garage-door-multi.groovy [hunk 2]
```diff
@@ -74,8 +64,8 @@ metadata {
 			state "battery", label:'${currentValue}% battery', unit:""
 		}
 
-		main(["status", "contact", "acceleration"])
-		details(["status", "contact", "acceleration", "temperature", "3axis", "battery"])
+		main(["contact", "acceleration"])
+		details(["contact", "contact", "acceleration", "temperature", "3axis", "battery"])
```
Comment (tpmanley) [inline devicetypes/smartthings/smartsense-garage-door-multi.src/smartsense-garage-door-multi.groovy]: `contact` is listed twice

File: devicetypes/smartthings/smartsense-garage-door-multi.src/smartsense-garage-door-multi.groovy [hunk 3]
```diff
@@ -48,15 +46,7 @@ metadata {
...
-		multiAttributeTile(name:"status", type: "generic", width: 6, height: 4){
-			tileAttribute ("device.status", key: "PRIMARY_CONTROL") {
-				attributeState "closed", label:'${name}', icon:"st.doors.garage.garage-closed", backgroundColor:"#00A0DC", nextState:"opening"
-				attributeState "open", label:'${name}', icon:"st.doors.garage.garage-open", backgroundColor:"#e86d13", nextState:"closing"
-				attributeState "opening", label:'${name}', icon:"st.doors.garage.garage-opening", backgroundColor:"#e86d13"
-				attributeState "closing", label:'${name}', icon:"st.doors.garage.garage-closing", backgroundColor:"#00A0DC"
-			}
-		}
-		standardTile("contact", "device.contact", width: 2, height: 2) {
+		standardTile("contact", "device.contact", width: 6, height: 4) {
```
Comment (tpmanley) [inline devicetypes/smartthings/smartsense-garage-door-multi.src/smartsense-garage-door-multi.groovy]: I think it should still be a `multiAttributeTile`

**Comment by greens** [comment]
updated

**Review by tpmanley — APPROVED** [review]

**Pull request merged**